Provide xsd tool example with `/classes` option (#42685)
dotnet/docs: docs/standard/serialization/xml-schema-def-tool-gen.md

2. Pass the XML Schema as an argument to the XML Schema Definition tool, which creates a set of classes that are precisely matched to the XML Schema, for example:  
  
    ```console  
    xsd mySchema.xsd /classes
    ```  

>     (The `/classes` option used in the above is to generate classes for the `mySchema` schema.)

Reviewer: We need to leave the blank line here:

```suggestion
    
    (The `/classes` option used in the above is to generate classes for the `mySchema` schema.)
```

2. Pass the XML Schema as an argument to the XML Schema Definition tool, which creates a set of classes that are precisely matched to the XML Schema, for example:  
  
    ```console  
    xsd mySchema.xsd /classes
    ```  

>     

Reviewer 2: ```suggestion

```

>     (The `/classes` option in this command is used to generate classes for the `mySchema` schema.)

Reviewer 2: ```suggestion

    (The `/classes` option in this command is used to generate classes for the `mySchema` schema.)
```

>     (The `/classes` option used in the above is to generate classes for the `mySchema` schema.)

Reviewer 2: ```suggestion
    (The `/classes` option in this command is used to generate classes for the `mySchema` schema.)
```

>     

Reviewer 2: ```suggestion

```

>     <br/>The tool can only process schemas that reference the World Wide Web Consortium XML specification of March 16, 2001. In other words, the XML Schema namespace must be `"http://www.w3.org/2001/XMLSchema"`, as shown in the following example.  

Reviewer 2: ```suggestion
    
    The tool can only process schemas that reference the World Wide Web Consortium XML specification of March 16, 2001. In other words, the XML Schema namespace must be `"http://www.w3.org/2001/XMLSchema"`, as shown in the following example.  
```
